app/events/dispatcher.py

```diff
@@ -0,0 +1,10 @@
+import logging
+from fastapi_events.dispatcher import dispatch as _dispatch
+
+logger = logging.getLogger(__file__)
+
+def dispatch(*args, **kwargs):
```

Comment:
I think this is already in develop? Maybe try doing `git pull origin develop` on your branch, I think some of these file changes will go away

Reply:
I'll do a merge